classes/requests/helpers/class-dintero-checkout-order.php

```diff
@@ -383,13 +399,17 @@ public function get_shipping_item( $shipping_item, $package_index = '' ) {
 			'type'            => 'shipping',
 		);
 
-		// Is this a pick up point? If the metadata exist, then it is.
+		// Is this a pick-up point? If it has metadata, then it is a pick-up point.
 		$encoded_meta = $shipping_item->get_meta( 'udc_delivery_data' );
 		if ( ! empty( $encoded_meta ) ) {
 			$meta = json_decode( $encoded_meta, true );
 
-			$id                               = $shipping_item->get_method_id() . ':' . $meta['id'];
-			$line_id                          = $id . ( $package_index !== '' ? ":{$package_index}" : '' );
+			if ( empty( $line_id ) ) {
+				// If the line_id is not set, we use the id as the line_id.
+				$id      = $shipping_item->get_method_id() . ':' . $meta['id'];
+				$line_id = $id . ( $package_index !== '' ? ":{$package_index}" : '' );
+			}
```

Comment:
Question: Same question as for shipping lines normally. Have we ensured that this is backwards compatible from testing as well?